src/configurations/schemes.conf

```diff
@@ -1585,3 +1585,147 @@ FallbackSchema=EditorColorSchema-cdbf28fc-1ddc-4d13-bb21-6a4043316a2f
 Underline_0=true
 Underline_18=true
 Underline_26=true
+
+[EditorColorSchema-cdbf28fc-1ddc-4d13-bb21-6a4043316a2f]
```

Comment:
This will replace the existing Dark theme.

Reply:
Sorry!

I used the UUID produced by QOWNNotes and didn't check for conflicts. I did begin with a copy of the Dark theme.

Reply:
Resolved in e638c93a6554b3b38e0509dc5db88d10808d2cca